src/server.js

```diff
@@ -0,0 +1,48 @@
+const cors = require("cors");
+const jwt = require("jsonwebtoken");
+require("dotenv/config");
+const express = require("express");
+const { ApolloServer, AuthenticationError } = require("apollo-server-express");
+const models = require("../models");
+
+const app = express();
+const port = 4000;
+app.use(cors());
+
+const typeDefs = require("./schema/");
+const resolvers = require("./resolvers/");
```

Comment:
personally, i like all my requires at the top of the file, and they also have a logic order:
1. Libraries
2. My own files (routes, models, middlewares, actions-components in the frontend)
3. css, if any